tests/k8s_testcases/example_policies/services1/services1-scheme.yaml

```diff
@@ -0,0 +1,91 @@
+namespaceList: ../../example_podlist/services1/ns_list.yaml
+podList: ../../example_podlist/services1/
+
+networkConfigList:
+  - name: ip
+    networkPolicyList:
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np1
+    networkPolicyList:
+      - services1-networkpolicy1.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np2
+    networkPolicyList:
+      - services1-networkpolicy2.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+queries:
+  - name: connectivity_map
+    connectivityMap:
```

Comment:
For connectivity map queries I suggest to add `expected_output` (which contains a path to an expected output file -- there are some examples on existing test schemes).